boards/FSM/Inc/Io/Io_Scancon2RMHF.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <stm32f3xx_hal.h>
+
+/**
+ * Initializes both the primary and secondary Scancon 2RMHF encoders
+ * @param htim_primary_encoder The timer handle for the primary encoder
+ * @param htim_secondary_encoder The timer handle for the secondary encoder
+ */
+void Io_Scancon2RMHF_Init(
+    TIM_HandleTypeDef *htim_primary_encoder,
+    TIM_HandleTypeDef *htim_secondary_encoder);
```

Comment:
So in the interests of keeping stuff modular, I think it might be better to have this "class" be for a single encoder, and then have an "AccelerationPedalEncoders" class that holds two of these encoders.

Reply:
Ok icic. What do you think is a good way to go about this? Should we have a separate Io files, one for the primary encoder and the other for the secondary encoder?

Reply:
Also just to make sure are you suggesting that we also create AccelerationPedalEncoders separate from App_AcceleratorPedal?

Reply:
> Ok icic. What do you think is a good way to go about this? Should we have a separate Io files, one for the primary encoder and the other for the secondary encoder?

Either option is fine with me.

> Also just to make sure are you suggesting that we also create AccelerationPedalEncoders separate from App_AcceleratorPedal?

Yep

Reply:
@garethellis0  Just giving this some thought and I was wondering why we want to separate AccelerationPedalEncoders from App_AcceleratorPedal?  I'm leaning more towards having a single App_AcceleratorPedal class (similar to what we did with App_Brake)

Reply:
That's also fine with me.

Reply:
Sounds good